packages/dev/core/src/AudioV2/webAudio/subNodes/webAudioAnalyzerSubNode.ts

```diff
@@ -0,0 +1,110 @@
+import type { Nullable } from "../../../types";
+import { _AudioAnalyzerSubNode } from "../../abstractAudio/subNodes/audioAnalyzerSubNode";
+import type { AudioAnalyzerFFTSizeType } from "../../abstractAudio/subProperties/abstractAudioAnalyzer";
+import type { _WebAudioEngine } from "../webAudioEngine";
+import type { IWebAudioInNode } from "../webAudioNode";
+
+/** @internal */
+export async function _CreateAudioAnalyzerSubNodeAsync(engine: _WebAudioEngine): Promise<_AudioAnalyzerSubNode> {
+    return new _WebAudioAnalyzerSubNode(engine);
+}
```

Comment:
This is async so dynamic imports can be used later if needed. I think it's important to keep it async for now, even if it's unnecessary, so users get used to the async nature of subnode initialization.

Note that removing `async` here will only push the issue further up the call stack, all the way to the public API for properties backed by subnodes, so removing `async` here will not be immediately helpful.

Reply:
@RaananW Using dynamic imports for the `_Create...SubNodeAsync` functions moves **`1.7 kB`** from main.js to async.js when using the generateFileSizes.js script. Is that enough to make it worth doing?

Reply:
1.7kb will make no improvement, unless there are further dependencies planned in the 1.7 kb of code (i.e. - some form of expansion of the feature)

Reply:
What do you think about leaving it as an `async` function anyway? ...so we can use dynamic imports later if needed without needing a breaking change to the public API.

Reply:
I think for now I'll leave it as-is and if we decide later that this function should not be async, then I'll change it in a separate PR along with the other async `_Create...SubNodeAsync` functions.